Reminder-App/Reminder-App/ReminderDetailEditDataSource.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  ReminderDetailEditDataSource.swift
+//  Reminder-App
+//
+//  Created by BrainX Technologies 11 on 16/07/2021.
+//
+
+import UIKit
+
+class ReminderDetailEditDataSource: NSObject {
+    
+    enum ReminderSection: Int, CaseIterable {
+        case title
+        case dueDate
+        case notes
+        
+        func cellIdentifier(for row: Int) -> String {
+            switch self {
+            case .title:
+                return "EditTitleCell"
+            case .dueDate:
+                return row == 0 ? "EditDateLabelCell" : "EditDateCell"
+            case .notes:
+                return "EditNotesCell"
+            }
+        }
```

Comment:
Create an extension of UITableViewCell and add a variable
 named reuseIdentifier
like
    static var reuseIdentifier: String {
        return String(describing: self)
    }